openedx/stanford/cms/envs/aws.py

```diff
@@ -13,6 +13,15 @@
     DEFAULT_COURSE_ABOUT_IMAGE_URL
 )
 EXTRA_MIMETYPES = ENV_TOKENS.get('EXTRA_MIMETYPES', EXTRA_MIMETYPES)
+FEATURES.update({
```

Comment:
Start fetching the value from the serialized JSON.

Reply:
Actually, this is superfluous...

The entire `FEATURES` dict gets copied over anyway, so we don't need to manually manipulate it via JSON..